R/Report_KRI.R

```diff
@@ -49,11 +49,9 @@ Report_KRI <- function(
   rlang::check_installed("knitr", reason = "to run `Report_KRI()`")
   rlang::check_installed("kableExtra", reason = "to run `Report_KRI()`")
 
-  # run report in temp directory
-  tpath <- fs::path_temp()
-
-  # specify strOutputFir path, depending on write access to strOutputDir
+  # specify strOutputDir path, depending on write access to strOutputDir
```

Comment:
I recommend moving all of this to `render_rmd`, and then having it take `strOutputFile` and `strOutputDir` arguments.